docs/gallery/decision_support/3_production_planning/index.md

```diff
@@ -1,17 +1,27 @@
 ---
 title: Production Planning
 category: decision_support
-data-keywords: optimization scenario cycle comparison
+data-keywords: optimization scenario cycle comparison enterprise
 short-description: Simulate manufacturing scenarios and optimize production quantities to reduce production costs.
 img: 3_production_planning/images/production-planning-data-visualization.png
 ---
-Production planning is pivotal in manufacturing, optimizing resources to meet
-demand while minimizing costs. This demo offers a solution for efficient production and cost management.
+Optimizing resources to meet demand while minimizing costs is pivotal in manufacturing. This is why 
+production planning tools are mandatory in manufacturing.
 
-[Try it live](https://production-planning.taipy.cloud/Data-Visualization){: .tp-btn target='blank' }
-[Get it on GitHub](https://github.com/Avaiga/demo-production-planning){: .tp-btn .tp-btn--accent target='blank' }
+!!! note "Enterprise application"
```

Comment:
I don't like this title.
Why not "Taipy Enterprise edition"?
Funny you don't even refer to that in the text body...